hbase-server/src/test/java/org/apache/hadoop/hbase/replication/ReplicationSourceDummyWithNoTermination.java

```diff
@@ -16,11 +16,16 @@
 package org.apache.hadoop.hbase.replication;
 
 public class ReplicationSourceDummyWithNoTermination extends ReplicationSourceDummy {
-
+  boolean firstTime = true;
+  boolean isTerminateInvoked = false;
```

Comment:
`isTerminateInvoked` is unused? nit: may want to make the other one volatile for thread-safety.